yos/exist.go

```diff
@@ -0,0 +1,77 @@
+package yos
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+)
+
+var (
+	ErrIsDir        = errors.New("target is a directory")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported var ErrIsDir should have comment or be unexported](https://app.codacy.com/app/an9an63/gut/pullRequest?prid=4635828)